src/main/java/com/hms_networks/sc/cumulocity/api/CConnectorApiMessageReader.java

```diff
@@ -716,7 +716,15 @@ public static void installFirmware_515(
               + ") from "
               + firmwareUrl);
       try {
-        downloadFirmwareFile(firmwareUrl);
+        // Perform firmware download and store result if the download/update was unsuccessful
```

Comment:
This comment makes it sound like a success should return.

Reply:
Updated comment. I agree that it was not described well enough what happens with successful firmware downloads.

Reply:
ok.. thanks. Approved!